assets/js/components/DashboardEntityApp.js

```diff
@@ -222,6 +222,9 @@ function DashboardEntityApp() {
 				<HelpMenu />
 			</Header>
 			<ConsentModeSetupCTAWidget />
+
+			<OverlayNotificationsRenderer />
```

Comment:
Not sure about the ordering here, Consent Mode stuff should be higher priority, right?

Reply:
It looks like rendering this component has already been addressed in `develop`. We should merge `develop` into this branch and accept incoming changes.

Reply:
In develop, the component rendering order doesn't matter because it uses a fixed position. However, after the refactoring, the notification is placed statically on mobile, hence the ordering should be near the header and belowe navbar.

Reply:
I've merged in develop to fix the merge conflict.